Source/WebKit/Shared/Cocoa/ArgumentCodersCocoa.mm

```diff
@@ -272,12 +275,23 @@ - (instancetype)initWithCGColor:(CGColorRef)color
 }
 #endif
 #endif
+#if USE(AVFOUNDATION)
+template<> Class getClass<AVOutputContext>()
+{
+    return PAL::getAVOutputContextClass();
+}
+#endif
+
 
 NSType typeFromObject(id object)
 {
     ASSERT(object);
 
     // Specific classes handled.
+#if USE(AVFOUNDATION)
+    if (PAL::isAVFoundationFrameworkAvailable() && [object isKindOfClass:[PAL::getAVOutputContextClass() class]])
```

Comment:
Again, I really don't think the last class does anything.

Reply:
Okay, see it now, was blind to it before.
Copied pattern from others.
You're right - cleared them all out